### PR TITLE
Add docstring for the `GammaRV` Op

### PR DESCRIPTION
**Pull request by rlouf**
In this PR I add a docstring for the `GammaRV` Op.

Here are a few important guidelines and requirements to check before your PR can be merged:
+ [x] There is an informative high-level description of the changes.
+ [NA] The description and/or commit message(s) references the relevant GitHub issue(s).
+ [x] [`pre-commit`](https://pre-commit.com/#installation) is installed and [set up](https://pre-commit.com/#3-install-the-git-hook-scripts).
+ [x] The commit messages follow [these guidelines](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+ [x] The commits correspond to [_relevant logical changes_](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes), and there are **no commits that fix changes introduced by other commits in the same branch/BR**.
+ [NA] There are tests covering the changes introduced in the PR.

### CODE REVIEW

**Comment by rlouf** [comment]
We can start simple with adding a single page to the documentation that contains doc for every distribution available, like PyMC does.

**Review by twiecki — APPROVED** [review]

**Comment by rlouf** [comment]
I added the `GammaRV` Op  to the [random.basic](https://aesara.readthedocs.io/en/latest/library/tensor/random/basic.html) documentation. Good to merge on my end.

**Review by brandonwillard — APPROVED** [review]

File: doc/library/tensor/random/basic.rst [hunk 1]
```diff
@@ -43,3 +43,11 @@ Reference
...
     This :class:`Op` is parameterized to draw numbers from many possible
     distributions.
+
+Distributions
+==============
+
+Aesara can produce :class:`RandomVariable`\s that draw samples from many different statistical distributions, using the following :class:`Op`\s.
+
+.. autoclass:: aesara.tensor.random.basic.GammaRV
+   :members: __call__
```
Comment (brandonwillard) [inline doc/library/tensor/random/basic.rst:53]: This is a good addition.  It reminds me that we still need to do this (i.e. replace manually entered method documentation with `autoclass` directives) for _all_ references to objects/classes in the documentation.

Comment (rlouf) [reply doc/library/tensor/random/basic.rst:53]: Yes. I looked at the documentation of the `aesara.tensor` module and was surprised to see it was all manually entered.

We'll also need to document the `RandomStream` approach better, and we can pull the corresponding Ops's docstrings.

Comment (brandonwillard) [reply doc/library/tensor/random/basic.rst:53]: We should also provide an `aesara.tensor.random.default_rng` interface to `RandomStream`.

**Pull request merged**